docs/maintain-guides-validator-payout.md

```diff
@@ -4,13 +4,26 @@ title: Validator Payout Overview
 sidebar_label: Validator Payout Overview
 ---
 
+## Era Points
+
+For every era (a period of time approximately 6 hours in length in Kusama, and 24 hours in Polkadot), validators are paid proportionally to the amount of _era points_ they have collected. Era points are reward points earned for payable actions like:
+
+- issuing validity statements for [parachain](learn-parachain) blocks.
+- producing a non-uncle block in the relay chain.
+- producing a reference to a previously unreferenced uncle block.
+- producing a referenced uncle block.
+
+_Note: An uncle block is a relay chain block that is valid in every regard, but which failed to become canonical. This can happen when two or more validators are block producers in a single slot, and the block produced by one validator reaches the majority of the network before the others. We call the lagging blocks uncle blocks._
```

Comment:
```suggestion
_Note: An uncle block is a relay chain block that is valid in every regard, but which failed to become canonical. This can happen when two or more validators are block producers in a single slot, and the block produced by one validator reaches the next block producer before the others. We call the lagging blocks uncle blocks._
```